api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java

```diff
@@ -526,6 +526,58 @@ public DataSetDefinition constructEriDSDDataset() {
             mappings),
         "");
 
+    dsd.addColumn(
+        "TN6",
+        "Total-N6: include all patients from 2",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "TN6",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries
+                        .getNumberOfPatientsOnArtAndAreMArkedInLastPuAsIorConFichaClinica(),
+                    mappings)),
+            mappings),
+        "");
+    addRow(
+        dsd,
+        "N6NE",
+        "N6: Number of active patients on ART (Non-pregnant and Non-Breastfeeding not on TB treatment) who are in PU and are Eligible",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N6N",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries
+                        .getNumberOfPatientsOnArtAndAreMArkedInLastPuAsIorConFichaClinicaAndEligible(),
+                    mappings)),
+            mappings),
+        dsdN6Disag());
+    dsd.addColumn(
+        "N6EA",
+        "DSD N6 stable Non-pregnant and Non-Breastfeeding Adults (>=15) NOT on TB treatment",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N6EA",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries
+                        .getNumberOfPatientsOnArtAndAreMArkedInLastPuAsIorConFichaClinicaAndEligibleAndNotOnTbtreatment(),
```

Comment:
Please replace `eriDSDCohortQueries.getNumberOfPatientsOnArtAndAreMArkedInLastPuAsIorConFichaClinicaAndEligibleAndNotOnTbtreatment()` with `eriDSDCohortQueries.getNumberOfPatientsOnArtAndAreMArkedInLastPuAsIorConFichaClinicaAndEligible()`

The method `EriDSDCohortQueries.getAllPatientsWhoAreActiveAndStable()` has already been updated to filter out patients on TB Treatment in the PR for **Ticket no. 287 - D1** (#153 )

Reply:
done